src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -1971,8 +2015,26 @@ private List<CompletionResult> GetResultForIdentifier(CompletionContext completi
             }
 
             TokenKind memberOperator = TokenKind.Unknown;
-            bool isMemberCompletion = (lastAst.Parent is MemberExpressionAst);
-            bool isStatic = isMemberCompletion && ((MemberExpressionAst)lastAst.Parent).Static;
+            bool isMemberCompletion = lastAst.Parent is MemberExpressionAst;
+            bool isStatic = false;
+            if (isMemberCompletion)
+            {
+                var currentExpression = (MemberExpressionAst)lastAst.Parent;
+                while (currentExpression.Extent.EndOffset >= completionContext.CursorPosition.Offset)
+                {
+                    if (currentExpression.Expression is MemberExpressionAst memberExpression && memberExpression.Member.Extent.EndOffset >= completionContext.CursorPosition.Offset)
```

Comment:
We could move this to the while condition.

Reply:
Done.